controllers/users_controllers.js

```diff
@@ -0,0 +1,9 @@
+const { fetchUsers } = require("../models/users_models.js");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers()
+    .then((users) => {
+      res.status(200).send(users);
```

Comment:
you should send users in a key of users, they shouldn't be free not organised in the body